src/main/scala/ai/privado/languageEngine/python/tagger/sink/AirflowOperatorSinkPass.scala

```diff
@@ -33,24 +33,32 @@ class AirflowOperatorSinkPass(cpg: Cpg, ruleCache: RuleCache) extends PrivadoPar
     }
   }
 
+  private def setCustomRuleToRuleCache(ruleInfoList: List[RuleInfo]): Unit = {
+    ruleInfoList.foreach(rule => {
+      ruleCache.setRuleInfo(rule)
+    })
+  }
+
   // Rule for Custom Operator
-  private def getCustomOperatorTag: RuleInfo = {
-    RuleInfo(
-      AIRFLOW_CUSTOM_RULE_ID,
-      "Airflow",
-      "Third Parties",
-      FilterProperty.METHOD_FULL_NAME,
-      Array(),
-      List("AirflowOperator"),
-      false,
-      "",
-      Map(),
-      NodeType.REGULAR,
-      "",
-      CatLevelOne.SINKS,
-      Constants.third_parties,
-      Language.PYTHON,
-      Array()
+  private def getCustomOperatorTag: List[RuleInfo] = {
+    List(
+      RuleInfo(
+        AIRFLOW_CUSTOM_RULE_ID,
+        "Airflow",
+        "",
+        FilterProperty.METHOD_FULL_NAME,
+        Array(),
+        List("AirflowOperator"),
+        false,
+        "",
+        Map(),
+        NodeType.REGULAR,
+        "",
+        CatLevelOne.SINKS,
+        Constants.third_parties,
+        Language.PYTHON,
+        Array()
+      )
```

Comment:
There is also a field `isGenerated` set that to `true`

Reply:
done.